workflow_scripts/tests/test_mutation_properties.py

```
@@ -9,9 +9,12 @@
 
 from __future__ import annotations
 
+import unittest.mock as mock
```

Ruff (0.15.21)

[warning] Use `from unittest import mock` in lieu of alias

Replace with `from unittest import mock`

(PLR0402)